src/main/java/ti4/commands2/player/SCPick.java

```diff
@@ -216,7 +220,7 @@ public static void secondHalfOfSCPick(GenericInteractionCreateEvent event, Playe
                 if (game.isShowBanners()) {
                     BannerGenerator.drawFactionBanner(privatePlayer);
                 }
-                MessageHelper.sendMessageToChannelWithButtons(privatePlayer.getPrivateChannel(), msgExtra + "\n Use Buttons to do turn.",
+                MessageHelper.sendMessageToChannelWithButtons(privatePlayer.getPrivateChannel(), "Use buttons to do turn.",
```

Comment:
Why msgExtra removed here? It's set in 207 but then not used here.

Reply:
I could be wrong, but I think that `msgExtra` is sent to the player on (new) line 212 with `MessageHelper.sendPrivateMessageToPlayer`, and so it was getting sent twice?